### PR TITLE
Convert to webpack first pass.

### PR DESCRIPTION
**Pull request by whscullin**
Basic smoke test works.

Todo:
- [x] Bring in missing ROM files
- [x] Production mode
- [x] Fix cpu tests.

### CODE REVIEW

File: js/main2.js [hunk 1]
```diff
@@ -362,18 +357,18 @@ case 'apple2':
...
+//     char_rom = apple2j_charset;
+//     break;
+// case 'apple2pig':
+//     rom = new Apple2ROM();
+//     char_rom = pigfont_charset;
+//     break;
+// case 'apple2lc':
+//     rom = new Apple2ROM();
+//     char_rom = apple2lc_charset;
+//     break;
```
Comment (iflan) [inline js/main2.js]: Why the commented out code?

File: js/main2e.js [hunk 2]
```diff
@@ -340,17 +336,18 @@ function openManage() {
...
-case 'apple2rm':
-    rom = new Apple2eEnhancedROM();
-    char_rom = rmfont_charset;
-    enhanced = true;
-    break;
+// case 'apple2rm':
+//     rom = new Apple2eEnhancedROM();
+//     char_rom = rmfont_charset;
+//     enhanced = true;
+//     break;
```
Comment (iflan) [inline js/main2e.js]: Same.

File: js/main2e.js [hunk 3]
```diff
@@ -126,7 +122,7 @@ var TAPE_TYPES = ['wav','aiff','aif','mp3','m4a'];
 
 var _currentDrive = 1;
 
-function openLoad(drive, event)
+window.openLoad = function(drive, event)
```
Comment (iflan) [inline js/main2e.js]: Are you meaning to set these on the global `window` object? If so, what's the advantage? Event listeners?

File: js/6502test.js [hunk 4]
```diff
@@ -4,7 +4,7 @@
 
 var LOG = false;
 
-function Test6502() {
+export default function Test6502() {
```
[the remark this answers is not in the document]
Comment (whscullin) [reply js/6502test.js]: They meant to be used in test.html. They need rewiring still

File: js/apple2io.js [hunk 5]
```diff
@@ -9,13 +9,10 @@
  * implied warranty.
  */
 
-/*globals debug: false */
-/*exported Apple2IO */
+import { debug } from './util';
 
-function Apple2IO(cpu, callbacks)
+export default function Apple2IO(cpu, callbacks)
```
[the remark this answers is not in the document]
Comment (whscullin) [reply js/apple2io.js]: I my general attitude is that if the only export is a single class, I don't bother naming it. It's just a personal preference, I can see either way.

File: js/main2.js (hunk 1 above)
Comment (whscullin) [reply js/main2.js]: I need to pull these files in still.

File: js/main2e.js (hunk 3 above)
Comment (whscullin) [reply js/main2e.js]: It results in the smallest delta at the moment.  I want to clean this up into a class in the near future, but this got things moving.

**Pull request merged**